mediaroute/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
This file can be deleted. The root `.gitignore` file already filters all build directories.